src/components/badge/Badge.jsx

```diff
@@ -1,12 +1,21 @@
 import './Badge.scss';
 import React from 'react';
 
+const BadgeColor = {
+  지인: 'BeigeBadge',
+  동료: 'PurpleBadge',
+  가족: 'GreenBadge',
+  친구: 'BlueBadge',
+};
+
+//전달받은 BadgeColor에 따라 뱃지의 className이 변경되도록 조건부 렌더링
 function Badge({ children }) {
+  const badgeClassName = `text-badge-container ${BadgeColor[children] || '친구'}`;
   return (
     <>
-      <span className='text-badge-container'>
+      <div className={badgeClassName}>
```

Comment:
```suggestion
<div className={classNames('text-badge-container', ${BadgeColor[children] || '친구'}) } >
```

classNames 라이브러리 사용해서 줄일 수 있을 거 같아용 ㅎㅎ
객체 사용해서 나눠주신거 좋아용😎

Reply:
확인했습니다! classNames 사용방법이 있었군요 ㅎㅎ 다음 PR때 변경해서 올리겠슴당 :)